shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -1182,13 +1219,23 @@ \subsection{Chain Transition}
         \end{array}\right)}
       }
       \\~\\
-      (\var{acnt}',~\var{pp}',~\var{ss}',~\var{ls}') \leteq \var{es}'
+      (\var{acnt}',~\var{ss}',~\var{ls}') \leteq \var{es}'
       &
-      (\var{us}',~(\var{ds}',~\var{ps'})) \leteq \var{ls}'
+      (\var{utxoSt}',~(\var{ds}',~\var{ps'}),~\var{us}'') \leteq \var{ls}'
       \\
-      \vdash\var{ps}'\trans{ocert}{bhb}\var{ps}''
-      &
-      \var{ls}'' \leteq (\var{us}',~(\var{ds}',~\var{ps}''))
+      \var{gkeys}
```

Comment:
I think `gkeys` also has to appear in `ChainEnv`

Reply:
ah, yes, in the rule definition, below the main line, it still has `pp_n`. I will change it to `gkeys` now, thanks!